polaris-core/src/main/java/org/apache/polaris/core/config/FeatureConfiguration.java

```diff
@@ -287,4 +287,25 @@ public static void enforceFeatureEnabledOrThrow(
                   + "This should only be set to 'true' for tests!")
           .defaultValue(false)
           .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Integer> EVENT_BUFFER_TIME_TO_FLUSH_IN_MS =
+          PolarisConfiguration.<Integer>builder()
+                  .key("EVENT_BUFFER_TIME_TO_FLUSH_IN_MS")
+                  .description("The minimum amount of time (in ms) before events in the buffer flush to the persistence")
+                  .defaultValue(30 * 1000) // Flush every 30s
+                  .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Integer> EVENT_BUFFER_NUM_SHARDS =
+          PolarisConfiguration.<Integer>builder()
+                  .key("EVENT_BUFFER_NUM_SHARDS")
+                  .description("The amount of shards available for event buffers")
+                  .defaultValue(5)
+                  .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Integer> EVENT_BUFFER_NUM_THREADS =
+          PolarisConfiguration.<Integer>builder()
+                  .key("EVENT_BUFFER_NUM_SHARDS")
+                  .description("The amount of threads available for processing event buffers")
```

Comment:
Are these specific to one event listener type, or a subset of them? If so, we should make a config object just for those.

Reply:
What's an event buffer implementation? From what I can see, it's one way to implement a `PolarisPersistingEventListener`, which itself is not a first-level concept and requires a specific eventlistener in the quarkus config to use. We don't use FeatureConfigurations for anything else that requires a quarkus config to select a type.